products/compute/api.yaml

```diff
@@ -3085,20 +3085,32 @@ objects:
           not enforced and the network behaves as if it did not exist. If this
           is unspecified, the firewall rule will be enabled.
         send_empty_value: true
-      - !ruby/object:Api::Type::NestedObject
-        name: 'logConfig'
+      - !ruby/object:Api::Type::Boolean
```

Comment:
`enableLogging` needs to be in a flattened `logConfig` otherwise the field gets flattened and expanded incorrectly. We'll have to get creative and maybe:

- make a second `logConfigLegacy` for `enableLogging` to be nested into. `logConfigLegacy` would be flattened and corrected to `logConfig` with `api_name` override

- or use the log config custom flatteners/expanders to set/pass the `enableLogging` flag

We can even do something like `conflicts` to make sure they aren't colliding

Reply:
ah, yes, i was hoping since `enableLogging` is at the root level in the API it wouldn't make a difference, but I think it's only that way for beta. i'm getting too far ahead of myself. thanks for the comments, i'll go back and dig deeper into this. thanks!

Reply:
Oh I didn't realize it was root level in beta, I just looked at `v1`! That makes sense then.